templates/azml-app/app/main.py

```diff
@@ -0,0 +1,60 @@
+from fastapi import FastAPI, Request
+import os
+import subprocess
+
+from app.generateScore import ScoreFileGenerator
+from app.startupHelpers import get_model_details, set_score_file_generator_vars
+
+app = FastAPI()
+
+# Global placeholders (will be initialized later)
+tokenizer = None
+model = None
+pipe = None
+
+# Serve out get traffic on root path ASAP, do not wait on model loading
+@app.get("/")
+def read_root():
+    return {"message": "Hello from FastAPI!"}
+
+@app.get("/readiness", status_code=200)
+def readiness():
+    """
+    Readiness probe for the application.
+    """
+    if tokenizer is None or model is None:
+        return {"status": "tokenizer or model hasn't been loaded yet"}, 503
```

Comment:
>return {"status": "tokenizer or model hasn't been loaded yet"}, 503 [](http://example.com/codeflow?start=8&length=67)

In error cases we should use something like:
raise HTTPException(status_code=503, detail="tokenizer or model hasn't been loaded yet")

[https://fastapi.tiangolo.com/tutorial/handling-errors/](https://fastapi.tiangolo.com/tutorial/handling-errors/)